startLumino.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+
+# How it works
+# ./startLumino <ENVIRONMENT_NAME>
+# This script is to start lumino using an environment called <ENVIRONMENT_NAME>.
+# ENVIRONMENT_NAME defaults to luminoNode
+
+# Here some explanation of the parameters:
+# ADDRESS_TO_USE: The address to set to be used by the node
+# PASSWORD_FILE_CONTENT: The password for the private key of the ADDRESS_TO_USE address.
+# TOKEN_NETWORK_REGISTRY: The token network registry contract address to be used by the node.
+# SECRET_REGISTRY: The secret registry contract address to be used by the node.
+# ENDPOINT_REGISTRY: The endpoint registry contract address to be used by the node.
+# HUB_MODE: Sets if the node will be a hub or not
+# RSK_ENDPOINT: The RSK endpoint url
+# EXPOSE_ENDPOINT: The expose url to use by the lumino node
+# PORT: The port number to setup for the lumino node
+# NETWORK_ID: The network id to connect to
+
+ENV_NAME=$1
+
+if [ "" == "${ENV_NAME}" ]; then
+  ENV_NAME=clientEnv
+fi
+
+KETYSTORE_PATH="/home/marcos/rsk/keystore"
```

Comment:
this doesn't look right

Reply:
this must be a script variable

Reply:
done

Reply:
it's good now. i would just leave names (e.g. `marcos`) out of the example but it is up to you